config/routes.rb

```diff
@@ -2,6 +2,9 @@
   resources :games do
     resources :groups
   end
+  resources :home do
+    post "select_game"
+  end
```

Comment:
Home is not a resource. To create a single route, you should do something like `post 'select_game', to: 'home#select_game', as: 'select_game'`

The first parameter specifies the actual URL, `to` specifies the controller, and `as` specifies the helper to generate (in this case, `select_game_path`)